packages/office-ui-fabric-react/src/components/Facepile/FacepileButton.tsx

```diff
@@ -11,8 +11,18 @@ export class FacepileButton extends BaseComponent<IButtonProps, {}> {
   protected _skipComponentRefResolution = true;
 
   public render(): JSX.Element {
-    const { theme, styles, className } = this.props;
+    const { theme, className, styles } = this.props;
 
-    return <BaseButton {...this.props} variantClassName={className} styles={getStyles(theme!, styles)} onRenderDescription={nullRender} />;
+    const customStyles = getStyles(theme!, className, styles);
+
+    return (
+      <BaseButton
+        {...this.props}
+        variantClassName="ms-Button--facepile"
+        className=""
```

Comment:
Why do we need `className=""`?

Reply:
BaseButtonClassNames styling has the className styles override the styles prop. If the className props is nonempty here, then any className styles (e.g. className="descriptiveOverflowButton") from the parent Facepile component will override any styles from the `styles` prop.

Reply:
You mean you try kill the `className` coming from the `{...this.props}` above it? If yes why not filter it out before returning and might as well filter out `styles` as you are overriding it anyway.